pyte/screens.py

```diff
@@ -886,9 +910,9 @@ def bell(self, *args):
 
     def alignment_display(self):
         """Fills screen with uppercase E's for screen focus and alignment."""
-        for line in self.buffer:
-            for column, char in enumerate(line):
-                line[column] = char._replace(data="E")
+        self.from_list(
+            [[Char(data="E", fg="default", bg="default") for _ in range(self.columns)] for _ in range(self.lines)]
```

Comment:
Do you have a reference for using "default" instead of preserving character attributes? VT220/VT520 only mention the Es.